Turning in Assingment

C1rcuit-Br3aker opened the pull request.

C1rcuit-Br3aker committed:
Starting assignment

C1rcuit-Br3aker committed:
starting assignment

C1rcuit-Br3aker committed:
add cssgram to stylesheets

C1rcuit-Br3aker committed:
add variables file

C1rcuit-Br3aker committed:
top nav markup

C1rcuit-Br3aker committed:
generate base styles

C1rcuit-Br3aker committed:
style top nav items

C1rcuit-Br3aker committed:
photographer image markup

C1rcuit-Br3aker committed:
style photographer logo

C1rcuit-Br3aker committed:
main content mark up

C1rcuit-Br3aker committed:
align icons, resize image

C1rcuit-Br3aker committed:
style image and author info

C1rcuit-Br3aker committed:
add max-width container

C1rcuit-Br3aker committed:
add outer container for article, add second image

C1rcuit-Br3aker committed:
add custom images

C1rcuit-Br3aker committed:
add filters, update image info

C1rcuit-Br3aker committed:
footer markup and basic style, realign main images

C1rcuit-Br3aker committed:
add author info above footer, add hover to footer images

C1rcuit-Br3aker committed:
add media query

C1rcuit-Br3aker committed:
resize filter for media query

C1rcuit-Br3aker committed:
adjust top margin of images, change top nav menu display on media query

C1rcuit-Br3aker committed:
change fa icon in profile image

C1rcuit-Br3aker committed:
adjust main article container width

rtablada commented:
Looks like there's some hard margins that are making it so that items aren't centering quite right on mobile.

<img width="652" alt="screen shot 2016-03-17 at 12 02 32 am" src="https://cloud.githubusercontent.com/assets/2532004/13836894/fb355456-ebd3-11e5-8c31-6762faa9f39d.png">

C1rcuit-Br3aker committed:
remove set margins and center content

C1rcuit-Br3aker committed:
add container for author info, move author container in article container

C1rcuit-Br3aker commented:
@rtablada Hard margins removed. I also added a container to the author info at the bottom and moved it inside the article container so that it lines up with the left side of the image when the page resizes.

rtablada commented:
@C1rcuit-Br3aker :+1:
